resource-sequence.puml

```diff
@@ -70,53 +72,69 @@ loop For each resource subscriber
     RIS -> DCB: POST /v1/resource-details/{resource-id}/definition (includes OVN)
 end
 
-== A new subscriber joins == 
 
-DCB2 -> DSS: PUT /DSS/v1/subscriptions/operation-subscription/
-DCB2 <-- DSS: operation references (empty)
+== OIM received request from operator and submits an operation to the DSS in proposed state == 
+op -> OIM: Op Request
+OIM -> DSS: PUT /dss/v1/operational_intent_references/{entityId}  (Proposed state)
+note over DSS
+Does not check any OVNs here.
+All operation intents accepted.
+end note
+OIM <-- DSS: subscribers to notify
+loop For all subscribers
+    OIM -> DCB: POST /dcb/v1/operational_intents (includes OVN)
+    DCB -> DCB: Adjust demand/capacity values
+    OIM -> DCB2:  POST /dcb/v1/operational_intents (includes OVN)
+    DCB2 -> DCB2: Adjust demand/capacity values
+end
 
+== A new subscriber joins == 
 DCB2 -> DSS: PUT /DSS/v1/subscriptions/resource-subscription/
 DCB2 <-- DSS: resource references
-
 loop For each resource reference received
     note over DCB2
     Get all resource details.
     end note
     DCB2 -> RIS: GET /v1/resource-details/{resource-id} (includes OVN)
     DCB2 <-- RIS: resource details (everything)
 end
-
-
-== OIM2 submits an operation to the DSS == 
-OIM2 -> DSS: PUT /dss/v1/operational_intent_references/{entityId}  (Proposed state)
-note over DSS
-Does not check any OVNs here.
-All operation intents accepted.
-end note
-OIM2 <-- DSS: subscribers to notify
-loop For all subscribers
-    ' OIM2 -> RIS: /psu/v1/operational_intents (includes OVN)
-    OIM2 -> DCB: /psu/v1/operational_intents (includes OVN)
-    DCB -> DCB: Adjust demand/capacity values
-    OIM2 -> DCB2:  /psu/v1/operational_intents (includes OVN)
-    DCB2 -> DCB2: Adjust demand/capacity values
+DCB2 -> DSS: PUT /DSS/v1/subscriptions/operation-subscription/
+DCB2 <-- DSS: operation references
+loop For each operation reference received
+    note over DCB2
+    Get all resource details.
+    end note
+    DCB2 -> OIM: GET /v1/operation_details/{operation-id} (includes OVN)
```

Comment:
In the X4 spec, both the GET and POST where PSUs were sharing operation details between each other were called operational_intents in the path.  We should keep the same consistency